contribs/gmf/src/controllers/bootstrap.js

```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
       withCredentials: false,
     },
   });
-  request.fail((jqXHR, textStatus) => {
-    window.alert(`Failed to get the dynamic: ${textStatus}`);
-  });
-  request.done((dynamic) => {
-    if (dynamic.doRedirect) {
-      const small_screen = window.matchMedia ? window.matchMedia('(max-width: 1024px)') : false;
-      if (small_screen && 'ontouchstart' in window) {
-        window.location.href = dynamic.redirectUrl;
+  request
+    .fail((jqXHR, textStatus) => {
+      window.alert(`Failed to get the dynamic: ${textStatus}`);
+    })
+    .done((dynamic) => {
+      if (dynamic.doRedirect) {
+        const small_screen = window.matchMedia ? window.matchMedia('(max-width: 1024px)') : false;
+        if (small_screen && 'ontouchstart' in window) {
+          window.location.href = dynamic.redirectUrl;
+        }
       }
-    }
 
-    if ('gmfCustomJavascriptUrl' in dynamic.constants) {
-      const script = document.createElement('script');
-      script.src = dynamic.constants.gmfCustomJavascriptUrl;
-      script.crossOrigin = 'anonymous';
-      document.body.append(script);
-    }
-    if ('gmfCustomStylesheetUrl' in dynamic.constants) {
-      const style = document.createElement('link');
-      style.href = dynamic.constants.gmfCustomStylesheetUrl;
-      style.rel = 'stylesheet';
-      style.crossOrigin = 'anonymous';
-      document.head.append(style);
-    }
+      if ('gmfCustomJavascriptUrl' in dynamic.constants) {
+        const gmfCustomJavascriptUrl = dynamic.constants.gmfCustomJavascriptUrl;
+        if (Array.isArray(gmfCustomJavascriptUrl)) {
+          gmfCustomJavascriptUrl.forEach((url) => {
+            addScript(url);
+          });
+        } else {
+          addScript(gmfCustomJavascriptUrl);
+        }
+      }
```

Comment:
Eventually a little bit more concise:

```suggestion
      if ('gmfCustomJavascriptUrl' in dynamic.constants) {
        const gmfCustomJavascriptUrl = Array.isArray(dynamic.constants.gmfCustomJavascriptUrl) ? dynamic.constants.gmfCustomJavascriptUrl : [dynamic.constants.gmfCustomJavascriptUrl];
        gmfCustomJavascriptUrl.forEach((url) => {
          addScript(url);
        });
      }
```